circuits/cpp/barretenberg/cpp/src/barretenberg/ecc/curves/bn254/fq.test.cpp

```diff
@@ -1,17 +1,24 @@
 #include "barretenberg/serialize/test_helper.hpp"
 #include "fq.hpp"
-#include "pseudorandom.hpp"
 #include <gtest/gtest.h>
 
 using namespace barretenberg;
 
-TEST(fq, msgpack)
+// used to ensure variables are evaluated at runtime and not compile time
```

Comment:
Why is it important that variables are evaluated at runtime?

Also `shallow_copy` might be a better term than `crude_copy` ?  Seems crude is being used to denote the fact that its a stack copy?

Reply:
I added a clarifying comment and renamed to shallow_copy 

The compiler can optimise out the EXPECT_EQ macro calls with compile-time parameters, which causes downstream problems in google test.